src/transformers/training_args.py

```diff
@@ -23,14 +23,12 @@
 from .trainer_utils import EvaluationStrategy
 from .utils import logging
 
-
```

Comment:
could we not change `training_args.py` in this PR and maybe only add changes that are necessary to make `run_mlm_flax.py` work?